src/DeploymentManager/DeploymentManager/help/Get-AzDeploymentManagerService.md

```diff
@@ -0,0 +1,216 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.DeploymentManager.dll-Help.xml
+Module Name: Az.DeploymentManager
+online version: https://docs.microsoft.com/en-us/powershell/module/az.deploymentmanager/get-azdeploymentmanagerservice
+schema: 2.0.0
+---
+
+# Get-AzDeploymentManagerService
+
+## SYNOPSIS
+Gets the service.
+
+## SYNTAX
+
+### Interactive (Default)
+```
+Get-AzDeploymentManagerService [-ResourceGroupName] <String> [-ServiceTopologyName] <String> [-Name] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ByServiceTopologyObject
+```
+Get-AzDeploymentManagerService [-ResourceGroupName] <String> [-Name] <String>
+ [-ServiceTopology] <PSServiceTopologyResource> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ByServiceTopologyResourceId
+```
+Get-AzDeploymentManagerService [-ResourceGroupName] <String> [-Name] <String>
+ [-ServiceTopologyResourceId] <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceId
+```
+Get-AzDeploymentManagerService [-ResourceId] <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### InputObject
+```
+Get-AzDeploymentManagerService [-Service] <PSServiceResource> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzDeploymentManagerService** cmdlet gets a service under a service topology, and returns an object that represents that service.
+Specify the service by its name, service topology it is in and the resource group name. Alternately, you can provide the Service object or the ResourceId.
+
+You can modify this object locally, and then apply changes to the service by using the Set-AzDeploymentManagerService cmdlet.
+{{Fill in the Description}}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzDeploymentManagerService -ResourceGroupName ContosoResourceGroup -ServiceTopologyName ContosoServiceTopology -Name ContosoService1
+```
+
+This command gets a service named ContosoService1 in a service topology named ContosoServiceTopology in the ContosoResourceGroup.
+
+### Example 2: Get a service using the resource identifier.
+```powershell
+PS C:\> Get-AzDeploymentManagerService -ResourceId "/subscriptions/subscriptionId/resourcegroups/ContosoResourceGroup/providers/Microsoft.DeploymentManager/serviceTopologies/ContosoServiceTopology/services/ContosoService1"
+```
+
+This command gets a service named ContosoService1 in a service topology named ContosoServiceTopology in the ContosoResourceGroup.
+
+### Example 3: Get a service using the service object.
+```powershell
+PS C:\> Get-AzDeploymentManagerService -Service $serviceObject
+PS C:\> {{ Add example code here }}
```

Comment:
@deveshguha please make sure all of these template snippets are removed from your markdown help files